libc/src/math/generic/CMakeLists.txt

```diff
@@ -747,6 +747,36 @@ add_entrypoint_object(
     -O3
 )
 
+add_entrypoint_object(
+  powf
+  SRCS
+    powf.cpp
+  HDRS
+    ../powf.h
+  DEPENDS
+    .common_constants
+    .explogxf
+    .exp2f
```

Comment:
is it possible to avoid these dependencies between entrypoints?

Reply:
Done.